lib/routes/ps/product.js

```diff
@@ -73,9 +73,9 @@ module.exports = async (ctx) => {
         url: `https://store.playstation.com/valkyrie-api/${langurl}/999/resolve/${gridName}?depth=2`,
     });
     const data = response.data;
-    const item = data.included && data.included.filter((item) => item.id === `${gridName}`);
+    const item = data.included?.filter((item) => item.id === gridName) ?? [];
     ctx.state.data = {
-        title: `PlayStation Store - ${item[0].attributes.name}`,
+        title: `PlayStation Store - ${item?.[0]?.attributes?.name ?? 'Unknown Name'}`,
```

Comment:
```suggestion
        title: `PlayStation Store - ${item[0].attributes?.name ?? 'Unknown Name'}`,
```

Unnecessary optional chain. `item` will always be an Array.